src/test/java/org/xstefank/TestUtils.java

```diff
@@ -0,0 +1,33 @@
+package org.xstefank;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import org.xstefank.model.yaml.FormatConfig;
+
+import java.io.File;
+import java.io.IOException;
+
+public class TestUtils {
+
+    public static JsonNode testPayload;
```

Comment:
if you have to make these public then at least make them final, we don't want first test to swap valid and invalid payloads